README.md

```diff
@@ -29,27 +29,36 @@ The Jupynium server will receive events from Neovim, keep the copy of the buffer
 ### Requirements
 
 - 💻 Linux, macOS and Windows (CMD, PowerShell, WSL2)
-- 🦊 Firefox (Other browsers are not supported due to their limitation with Selenium)
-- 🦎 Mozilla geckodriver (May be automatically installed with Firefox. Check `geckodriver -V`)
 - ✌️ Neovim >= v0.8
+- 🦊 Firefox
+  - Other browsers are not supported due to their limitation with Selenium
+- 🦎 Mozilla geckodriver
+  - May already be installed with Firefox. Check `geckodriver -V`
 - 🐍 Python >= 3.7
-- 📔 Jupyter Notebook >= 6.2 (Doesn't support Jupyter Lab)
+  - Supported Python installation methods include system-level and [miniconda](https://docs.conda.io/en/latest/miniconda.html)
+  - [Anaconda](https://www.anaconda.com/products/distribution) support is experimental
```

Comment:
I don't think you need to mention anaconda support is experimental and it only works on miniconda. This plugin has nothing to do with the environment and the problem you had was very environment specific. I just tested with anaconda in Linux and it works out of the box.